src/bindings/python/wheel/setup.py

```diff
@@ -426,6 +459,15 @@ def copy_package_data(self, src_dirs):
             packages.append(WHEEL_LIBS_PACKAGE)
             package_data.update({WHEEL_LIBS_PACKAGE: ["*"]})
 
+    def copy_package_data(self, src_dirs):
```

Comment:
we also need to ensure TBB headers, cmake files are part of wheel as well, because they can be used by OpenVINO extensions to speed-up some pieces of code.